File: app/src/main/java/tv/spacedentist/android/util/SDLoggerAndroid.java
package tv.spacedentist.android.util;

import android.util.Log;

public class SDLoggerAndroid implements SDLogger {

    public void i(String tag, String msg) {
        Log.i(tag, msg);
    }

    public void d(String tag, String msg) {
        Log.d(tag, msg);
    }

    public void e(String tag, String msg) {
        Log.e(tag, msg);
    }

    public void e(String tag, String msg, Throwable tr) {
        Log.e(tag, msg, tr);
    }
}

File: app/src/main/java/tv/spacedentist/android/util/SDLoggerAndroid.kt
package tv.spacedentist.android.util

import android.util.Log

class SDLoggerAndroid : SDLogger {

    override fun i(tag: String, msg: String) {
        Log.i(tag, msg)
    }

    override fun d(tag: String, msg: String) {
        Log.d(tag, msg)
    }

    override fun e(tag: String, msg: String) {
        Log.e(tag, msg)
    }

    override fun e(tag: String, msg: String, tr: Throwable) {
        Log.e(tag, msg, tr)
    }
}
